config/common.php

```diff
@@ -18,11 +28,37 @@
 }
 
 return [
+    LogCollectorInterface::class => LogCollector::class,
+    EventCollectorInterface::class => EventCollector::class,
+    CommonServiceCollectorInterface::class => CommonServiceCollector::class,
+    ContainerProxyInterface::class => ContainerProxy::class,
+    ContainerProxyConfig::class => static function (ContainerInterface $container) use ($params) {
+        $collector = $container->get(CommonServiceCollectorInterface::class);
+        $dispatcher = $container->get(EventDispatcherInterface::class);
+        $debuggerEnabled = (bool)($params['debugger.enabled'] ?? false);
+        $trackedServices = (array)($params['debugger.trackedServices'] ?? []);
+        $decoratedServices = (array)($params['container.decorators'] ?? []);
+        $path = $container->get(Aliases::class)->get("@runtime/cache/container-proxy");
+        if (!is_dir($path) && !mkdir($path)) {
```

Comment:
what about `recuresive=true` on `mkdir` call?
In my project in the `runtime` dirrectory by default the `cache` dir is absent

Reply:
Done